libs/blocks/caas-marquee/caas-marquee.js

```diff
@@ -112,6 +133,7 @@ function getMetadata(el) {
 function isProd() {
   const { host } = window.location;
   return !(host.includes('hlx.page')
+    || host.includes('hlx.live')
     || host.includes('localhost')
     || host.includes('stage.adobe')
     || host.includes('corp.adobe'));
```

Comment:
I would go with something more specific here:
```
|| host.endsWith('.stage.adobe.com')
|| host.endsWith('.corp.adobe.com')
```

Reply:
Done 👍

Reply:
Actually changing this back as it's failing CodeQL:

![image](https://github.com/adobecom/milo/assets/10876964/37affdc3-452f-4bea-8e69-4953bd7d5e32)